strawberryfields/apps/qchem/dynamics.py

```diff
@@ -250,3 +254,61 @@ def prob(samples: list, excited_state: list) -> float:
         raise ValueError("The excited state must not contain negative values")
 
     return samples.count(excited_state) / len(samples)
+
+
+def marginal(mu: np.ndarray, V: np.ndarray, n_max: int, hbar: float = 2.0) -> np.ndarray:
```

Comment:
What do other code-reviewers think of changing the name to `marginals` with an s?

Reply:
if it can do more than one mode then `marginals` sounds better.